js/localise-times.js

```diff
@@ -0,0 +1,51 @@
+function localise_times(site_timezone) {
+  let options = {
+    day: 'numeric',
+    month: 'long',
+    year: 'numeric',
+    hour: 'numeric',
+    hour12: true,
+    minute: 'numeric',
+    dayPeriod: 'short',
+  };
+  const date_formatter_with_minutes = Intl.DateTimeFormat(undefined, options);
+  delete options.minute;
+  const date_formatter_without_minutes = Intl.DateTimeFormat(undefined, options);
+
+  if (date_formatter_with_minutes.resolvedOptions().timeZone == site_timezone) {
+    return;
+  }
+
+  document.querySelectorAll("time.time-span").forEach((elem) => {
+    const datetime = elem.parentElement.attributes.datetime.value;
```

Comment:
```suggestion
    const datetime = elem.attributes.datetime.value;
```

Without this change I get the following errors in Chrome and Firefox.

![image](https://user-images.githubusercontent.com/1496834/100023539-0f246d80-2ddd-11eb-8beb-adcd15d11215.png)
![image](https://user-images.githubusercontent.com/1496834/100023636-3c711b80-2ddd-11eb-99df-58c615cdb506.png)

Reply:
D'oh. This is a result of some refactoring I did.